sql/moz-fx-data-shared-prod/firefox_ios/ltv_states/view.sql

```diff
@@ -0,0 +1,28 @@
+{% set max_weeks = 32 %}
+{% set death_time = 160 %}
+{% set lookback = 28 %}
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_ios.ltv_states`
+AS
+SELECT
```

Comment:
I think this view is missing the actual state, right? i.e. what is the state of this user on this day

Reply:
Yeah that's a good point, let me add that in as a new column